packages/date-picker/test/basic.test.js

```diff
@@ -421,38 +421,6 @@ describe('basic features', () => {
       datepicker.max = '2016-12-31';
     });
 
-    it('should be invalid when value is out of limits', () => {
-      datepicker.value = '2017-01-01';
-      expect(datepicker.invalid).to.be.equal(true);
-    });
-
-    it('should be valid when value is inside the limits', () => {
-      datepicker.value = '2016-07-14';
-      expect(datepicker.invalid).to.be.equal(false);
-    });
-
-    it('should be valid when value is the same as min date', () => {
-      datepicker.value = '2016-01-01';
-      expect(datepicker.invalid).to.be.equal(false);
-    });
-
-    it('should be valid when value is the same as max date', () => {
-      datepicker.value = '2016-12-31';
-      expect(datepicker.invalid).to.be.equal(false);
-    });
```

Comment:
**note:** This is now tested in `form-input.test.js`, see:

- [the `min` section](https://github.com/vaadin/web-components/pull/4163/files#diff-746ffa5fcc10c6bab313c794447622d931d27c361bb3c354f0135efabf98046fR226)
- [the `max` section](https://github.com/vaadin/web-components/pull/4163/files#diff-746ffa5fcc10c6bab313c794447622d931d27c361bb3c354f0135efabf98046fR276)